src/credentials/sso/accessToken.ts

```diff
@@ -0,0 +1,11 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface AccessToken {
```

Comment:
interface is pretty self explanatory, but should probably have a comment about what it does (is this to map a service response or etc). Also would slightly prefer something like `SsoAccessToken`

Reply:
Done